client/state/user-devices/reducer.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import {
+	USER_DEVICES_ADD,
+} from 'state/action-types';
+
+export default createReducer( {}, {
+	[ USER_DEVICES_ADD ]: ( state, { devices } ) => devices
```

Comment:
The argument is very compelling. I've changed the reducer in 69458d6cdd1929af9dcb9065b3a52beb9ef9bed3, but I'm keeping `createReducer`. Should I use `createReducerWithPersistance` instead?